source/_pages/docs/getting_started/compile.ngt

```diff
@@ -37,5 +37,5 @@ $ mvn clean package
 </p>
 
 <pre><code language="bash">
-$ java -cp heroic-dist/target/heroic-dist-0.0.1-SNAPSHOT.jar com.spotify.heroic.HeroicService &lt;config&gt;
+$ java -cp heroic-dist/target/heroic-dist-0.0.1-SNAPSHOT-shaded.jar com.spotify.heroic.HeroicService &lt;config&gt;
```

Comment:
I have messed a bit with commits and branches. Now request about shaded version is in separate pull request https://github.com/spotify/heroic/pull/38.
But I have not get your comment actually.
I was following documentation and it is impossible to start Heroic with provided command without additional arguments or environment configuration. Provided fix make it possible to run  application according to documentation.